2019-11-04      Gaius Mulley <gaius.mulley@southwales.ac.uk>

	* gm2/Make-lang.in:  (gm2.all.cross) added
          plugin/m2rte$(exeext).so as a dependency.

diff --git a/gcc-versionno/gcc/gm2/Make-lang.in b/gcc-versionno/gcc/gm2/Make-lang.in
index 8c7a9b5a..88c332b0 100644
--- a/src/gcc/gm2/Make-lang.in
+++ b/src/gcc/gm2/Make-lang.in
@@ -156,7 +156,7 @@ GM2_LIB_DIR_MIN= $(DESTDIR)$(libsubdir)/gm2/min
 
 # Build hooks:
 
-gm2.all.cross: xgm2-cross$(exeext)
+gm2.all.cross: xgm2-cross$(exeext) plugin/m2rte$(exeext).so
 
 gm2.start.encap: xgm2$(exeext) plugin/m2rte$(exeext).so $(GM2_LINK_TOOLS)
 gm2.rest.encap:
